ui/src/table.test.tsx

```diff
@@ -77,6 +77,91 @@ describe('Table.tsx', () => {
     expect(getAllByRole('gridcell')[0].textContent).toBe('6/23/2022, 12:50:28 AM')
   })
 
+  it('Renders tags correctly', () => {
+    tableProps = {
+      ...tableProps,
+      columns: [
+        { name: 'colname1', label: 'col1' },
+        {
+          name: 'colname2',
+          label: 'col2',
+          filterable: true,
+          cell_type: {
+            tag: {
+              name: 'tags',
+              tags: [
+                { label: 'TAG1', color: 'red' },
+                { label: 'TAG2', color: 'green' },
+                { label: 'TAG3', color: 'blue' },
+              ]
+            }
+          }
+        },
```

Comment:
Can be extracted into a separate variable so that it can be reused for both tests.